rrtmgp/mo_gas_optics_rrtmgp.F90

```diff
@@ -501,6 +503,7 @@ function compute_gas_taus(this,                       &
     ! Compute dry air column amounts (number of molecule per cm^2) if user hasn't provided them
     !
     idx_h2o = string_loc_in_array('h2o', this%gas_names)
+    col_dry_wk => col_dry_arr
```

Comment:
Not that it matters, since pointer assignment is cheap, but I wonder why you do this since you might re-assign the pointer just below.

Reply:
Because, in the original code result in a crush when was run on GPU with error message "Null pointer for col_dry_wk". I was trying to nullify it but with no success. That was the simplest solution and I am still puzzled by the problem.